model/org.eclipse.birt.report.model/src/org/eclipse/birt/report/model/api/DesignElementHandle.java

```diff
@@ -2205,7 +2196,7 @@ public List paste(String propName, DesignElementHandle content) throws SemanticE
 	 * @return a list containing all errors for the pasted element
 	 * @throws SemanticException if the element is not allowed to paste
 	 */
-	public List paste(String propName, IDesignElement content) throws SemanticException {
+	public List<?> paste(String propName, IDesignElement content) throws SemanticException {
```

Comment:
All these paste method return a result like that of `List<ErrorDetail> convertExceptionList` so this return type seems unhelpful.

Reply:
I will try it and I understand that you reference to the change of "convertExceptionList()".
But at my testing I got conflicts with the methods which are using the "paste()", so I will see after the change.

Reply:
Change is done.